Leetcode/EigenDemo/eigenPractice.hpp

```diff
@@ -0,0 +1,28 @@
+//
+//  eigenPractice.hpp
+//  EigenDemo
+//
+//  Created by dsh on 2021/7/18.
+//
+
+#ifndef eigenPractice_hpp
+#define eigenPractice_hpp
+
+#include <stdio.h>
+#include <iostream>
+
+#include "Eigen/core"
+#include "Eigen/Dense"
+
+using namespace Eigen;
+
+#define MATRIX_SIZE 50
+
+class eigenPractice {
+public:
+    eigenPractice() {};
+    void eigenBasic();
+    void eigenGeometry();
```

Comment:
```suggestion
    void EigenBasic();
    void EigenGeometry();
```

https://google.github.io/styleguide/cppguide.html#Function_Names